packages/client/src/generation/TSClient/Model.ts

```diff
@@ -48,7 +48,7 @@ export class Model implements Generatable {
   constructor(protected readonly model: DMMF.Model, protected readonly context: GenerateContext) {
     this.dmmf = context.dmmf
     this.genericsInfo = context.genericArgsInfo
-    this.type = this.context.dmmf.outputTypeMap[model.name]
+    this.type = this.context.dmmf.outputTypeMap.model[model.name]
```

Comment:
Can be model or composite, right?

Reply:
Actually, on the `schema` level of DMMF we don't differentiate between models and composites - derived types for both land here.